packages/test/test-end-to-end-tests/src/test/deRehydrateContainerTests.spec.ts

```diff
@@ -585,14 +585,40 @@ describeCompat(
 					await defaultDataStoreBefore.getSharedObject<SharedString>(sharedStringId);
 				const intervalsBefore = sharedStringBefore.getIntervalCollection("intervals");
 				sharedStringBefore.insertText(0, "Hello");
-				let interval0: SequenceInterval | undefined = intervalsBefore.add({
-					start: 0,
-					end: 0,
-				});
-				let interval1: SequenceInterval | undefined = intervalsBefore.add({
-					start: 0,
-					end: 1,
-				});
+				let interval0: SequenceInterval | undefined;
```

Comment:
this test is another good example of places where the old patterns for writing tests resulted in a reduced compat matrix. Since it uses a custom loader, compat options which claim to be testing different versions of SharedString were actually only ever testing the latest version. Using the `apis` argument forced changes here to account for changes to IntervalCollection's `add` and `change` API.

(also gave me a nice thing to point the README to for example of back-compat code)